### PR TITLE
Fixing pagination tests for CRAN by specifying `hsep = "-"`

### LINKED ISSUE #117: URGENT! Failed CRAN Checks: 2023-04-17
CRAN checks for rlistings resulted in one or more (ERROR)s:

See the table below for a summary of the checks run by CRAN:

|Flavor                            |Version | Tinstall| Tcheck| Ttotal|Status |Flags |
|:---------------------------------|:-------|--------:|------:|------:|:------|:-----|
|r-devel-linux-x86_64-debian-clang |0.2.1   |     7.96| 187.74| 195.70|OK     |NA    |
|r-devel-linux-x86_64-debian-gcc   |0.2.1   |     6.04| 130.61| 136.65|OK     |NA    |
|r-devel-linux-x86_64-fedora-clang |0.2.1   |       NA|     NA| 236.30|OK     |NA    |
|r-devel-linux-x86_64-fedora-gcc   |0.2.1   |       NA|     NA| 240.89|OK     |NA    |
|r-devel-macos-arm64               |0.2.1   |       NA|     NA|  68.00|OK     |NA    |
|r-devel-macos-x86_64              |0.2.1   |       NA|     NA| 144.00|OK     |NA    |
|r-devel-windows-x86_64            |0.2.1   |    10.00| 329.00| 339.00|OK     |NA    |
|r-patched-linux-x86_64            |0.2.1   |     6.16| 175.21| 181.37|OK     |NA    |
|r-release-linux-x86_64            |0.2.1   |     7.05| 174.36| 181.41|OK     |NA    |
|r-release-macos-arm64             |0.2.1   |       NA|     NA|  56.00|OK     |NA    |
|r-release-macos-x86_64            |0.2.1   |       NA|     NA|  83.00|OK     |NA    |
|r-release-windows-x86_64          |0.2.1   |    10.00| 321.00| 331.00|OK     |NA    |
|r-oldrel-macos-arm64              |0.2.1   |       NA|     NA|  54.00|OK     |NA    |
|r-oldrel-macos-x86_64             |0.2.1   |       NA|     NA|  86.00|OK     |NA    |
|r-oldrel-windows-ix86+x86_64      |0.2.1   |    16.00| 293.00| 309.00|ERROR  |NA    |

All details and logs are available here: https://cran.r-project.org/web/checks/check_results_rlistings.html

### PR DESCRIPTION
**Pull request by Melkiades**
Fixes #117

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 ### Bug fixes
  * `matrix_form(lsting, TRUE)` no longer throws an error.
+ * Fixed pagination tests to have compatibility with non-`UTF8` coding by specifing `hsep = "-"`.
```
Comment (Melkiades) [inline NEWS.md:22]:
```suggestion
 * Fixed pagination tests to have compatibility with non-`UTF8` coding by specifying `hsep = "-"`.
```

File: NEWS.md [hunk 2]
```diff
@@ -19,6 +19,7 @@
 
 ### Bug fixes
  * `matrix_form(lsting, TRUE)` no longer throws an error.
+ * Fixed pagination tests to have compatibility with non-`UTF8` coding by specifying `hsep = "-"`.
```
Comment (gmbecker) [inline NEWS.md:22]: This doesn't warrant a NEWS item, please remove.

**Review by gmbecker — CHANGES_REQUESTED** [review]
Missing similar changes in `test-print.R`

Use https://win-builder.r-project.org/ (oldrel option) to confirm full fix before pushing again please.

File: NEWS.md (hunk 2 above)
Comment (Melkiades) [reply NEWS.md:22]: I think this is quite a definition of the section called "Bug fixes", but it will be anyway squashed in the future

**Comment by Melkiades** [comment]
> Missing similar changes in `test-print.R`
> 
> Use https://win-builder.r-project.org/ (oldrel option) to confirm full fix before pushing again please.

I tried to check against old release but I may have sent you the results

**Comment by Melkiades** [comment]
> Missing similar changes in `test-print.R`
> 
> Use https://win-builder.r-project.org/ (oldrel option) to confirm full fix before pushing again please.

I tried to check against old release but I may have sent you the results.

When I try to do it with my email an error happens:
```r
> devtools::check_win_oldrelease(email = "davide.garolini@x.com")
Building windows version of rlistings (0.2.1.9001)
ℹ Using R-oldrelease with win-builder.r-project.org.
Email results to davide.garolini@x.com?

1: For sure
2: Nope
3: Not yet

Selection: 1
── R CMD build ──────────────────────────────────────────────────────────────────────────────────────────
✔  checking for file ‘/home/garolind/NEST/rlistings/DESCRIPTION’ ...
─  preparing ‘rlistings’:
✔  checking DESCRIPTION meta-information ...
─  installing the package to build vignettes
✔  creating vignettes (5.2s)
─  checking for LF line-endings in source and make files and shell scripts
─  checking for empty or unneeded directories
─  building ‘rlistings_0.2.1.9001.tar.gz’
   
Error in curl::curl_fetch_memory(url, handle = h) : 
  Failed FTP upload: 550
```

**Review by shajoezhu — APPROVED** [review]
Thanks @Melkiades for fixing this. This looks good to me. I think we can move forward on this. Cheers

**Comment by shajoezhu** [comment]
Hi @gmbecker , can you please review this PR and unblock the merge. Thanks

**Comment by Melkiades** [comment]
can we go with this @gmbecker

**Pull request merged**